src/components/landing/Footer.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Footer() {
+  return (
+    <div className="flex items-center py-8 px-4 md:px-8 lg:px-0 bg-other-background">
```

Comment:
You can remove `bg-other-background` since you have a parent `body` element that contains this property already.